src/Entity/Snowboarder.php

```diff
@@ -159,21 +212,23 @@ public function setFirstName(string $firstName): self
     }
 
     /**
-     * @return string|null
+     * The visual identifier that represents this user.
+     *
+     * @see UserInterface
      */
-    public function getPseudo(): ?string
+    public function getUsername(): string
     {
-        return $this->pseudo;
+        return (string) $this->username;
```

Comment:
```suggestion
        return $this->username;
```

You don't have to cast string here. Because setUsername is typed string.